src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponent.scala

```diff
@@ -115,9 +115,15 @@ trait ClusterComponent extends LeoComponent {
       }
     }
 
-    def listByLabels(labelMap: Map[String, String]): DBIO[Seq[Cluster]] = {
-      val query = if (labelMap.isEmpty) {
+    def listByLabels(labelMap: Map[String, String], includeDeleted: Boolean): DBIO[Seq[Cluster]] = {
+      val clusterStatusQuery = if (includeDeleted)
         clusterQueryWithLabels
+      else
+        clusterQueryWithLabels.filter { _._1.status inSetBind ClusterStatus.existingStatuses.map(_.toString) }
```

Comment:
What about `filterNot { status === 'Deleted' }` instead?  Do we really need `existingStatuses`?